bliss/api.py

```diff
@@ -151,18 +153,9 @@ def predict_sdss(
                 catalog as an astropy table, and probabilistic predictions catalogs as astropy
                 tables
         """
-        cfg = OmegaConf.create(self.base_cfg)
-        # apply overrides
-        cfg.predict.weight_save_path = cfg.paths.output + f"/{weight_save_path}"
-        for k, v in kwargs.items():
-            OmegaConf.update(cfg, k, v)
-        est_cat, _, _, _, pred_for_image_id = _predict(cfg)
-        est_cat_table = fullcat_to_astropy_table(est_cat)
-        pred_tables = {}  # indexed by image_id
-        for image_id, pred in pred_for_image_id.items():
-            pred_tables[image_id] = pred_to_astropy_table(pred)
-        return est_cat, est_cat_table, pred_tables
+        return self.predict("sdss", weight_save_path, **kwargs)
 
+    @deprecated(deprecated_in_version="0.3.1", message="Use the general predict function instead")
```

Comment:
At this stage, better to just delete it; our codebase is still very much in flux and our code is easier to understand if there's less of it. Let's start worrying about backward compatibility once we release v1.0.

Reply:
Not a problem!